packages/nc-gui/components/smartsheet/toolbar/FilterInput.vue

```diff
@@ -0,0 +1,90 @@
+<script setup lang="ts">
+import type { ColumnType } from 'nocodb-sdk'
+import { useColumn } from '~/composables/useColumn'
+import { ColumnInj, EditModeInj, computed, provide, toRef } from '#imports'
+import type { Filter } from '~/lib'
+import SingleSelect from '~/components/cell/SingleSelect'
```

Comment:
For some reason explicit imports from '#components' doesn't work here, so had to list that ugly